templates/jaasai/experts.html

```diff
@@ -32,9 +32,9 @@ <h2 class="expert__title">
     </div>
   </div>
   <div class="row">
-    <div class="u-equal-height">
-      {% for expert_id, expert in context.experts.items() %}
-        <div class="col-4 p-card">
+    {% for expert_id, expert in context.experts.items() %}
```

Comment:
`expert_id` doesn't appear to be used.

Reply:
It's being used to load the expert logo images.

Reply:
Are you sure? It's not used in any code in this loop...

Reply:
Remove it and see for yourself.

Reply:
You might be able to replace `expert_id` with an `_` if you don't need to use the value.